modules/fx/arrow-fx/src/main/kotlin/arrow/fx/IORunLoop.kt

```diff
@@ -12,22 +9,35 @@ import kotlin.coroutines.CoroutineContext
 import kotlin.coroutines.EmptyCoroutineContext
 import kotlin.coroutines.startCoroutine
 
+sealed class IOResult<out E, out A> {
+  data class Right<A>(val value: A) : IOResult<Nothing, A>()
```

Comment:
We either use left-right analogy or success-failure-error. Now that Try is out those names are free.

Reply:
Yes, this should probably be `Success` in analogy to success-failure-error`.